austation/code/modules/mob/living/emote.dm

```diff
@@ -9,3 +9,20 @@
 
 /datum/emote/living/snap
 	cooldown = (5 SECONDS)
+
+/datum/emote/living/surrender/run_emote(mob/user, params, type_override, intentional)
+	..()
+	var/mob/living/L = user
+	var/atom/A = L
+	var/image/I
+	if(L.surrendered)
+		I = image('austation/icons/mob/actions/actions.dmi', A, "surrender", A.layer+1)
+		L.apply_status_effect(STATUS_EFFECT_PARALYZED)
```

Comment:
this permanent paralyze has me a little concerned; the only way out of it is to use the surrender emote again, or be handcuffed. I really think it would be better to have a timed paralyze for about 15 seconds or so before you get up automatically.